spec/features/journeys/happy_paths/previously_received_targeted_delivery_funding_spec.rb

```diff
@@ -119,7 +120,7 @@ def run_scenario(js:)
       "cohort_id" => Cohort.current.id,
       "course_id" => Course.find_by(identifier: "npq-senior-leadership").id,
       "schedule_id" => nil,
-      "ecf_id" => nil,
+      "ecf_id" => Application.last.ecf_id,
```

Comment:
will this be flakey? can we get a more accurate application here and everywhere else?

Reply:
Ah yes, will fix those